src/Arcus.EventGrid.Tests.Integration/Publishing/EventGridEventPublishingTests.cs

```diff
@@ -4,164 +4,102 @@
             // Arrange
-            var topicEndpoint = Configuration.GetValue<string>("Arcus:EventGrid:TopicEndpoint");
-            var endpointKey = Configuration.GetValue<string>("Arcus:EventGrid:EndpointKey");
             const string eventSubject = "integration-test";
             const string licensePlate = "1-TOM-337";
             var eventId = Guid.NewGuid().ToString();
             var @event = new NewCarRegistered(eventId, eventSubject, licensePlate);
 
-            // Act
-            await EventGridPublisherBuilder
-                .ForTopic(topicEndpoint)
-                .UsingAuthenticationKey(endpointKey)
-                .Build()
-                .PublishAsync(@event);
+            IEventGridPublisher publisher = _endpoint.BuildPublisher();
 
+            // Act
+            await publisher.PublishAsync(@event);
             TracePublishedEvent(eventId, @event);
 
             // Assert
-            var receivedEvent = _serviceBusEventConsumerHost.GetReceivedEvent(eventId);
+            var receivedEvent = _endpoint.ServiceBusEventConsumerHost.GetReceivedEvent(eventId);
             AssertReceivedNewCarRegisteredEvent(eventId, @event.EventType, eventSubject, licensePlate, receivedEvent);
         }
 
         [Fact]
         public async Task PublishSingleRawEvent_WithBuilder_ValidParameters_Succeeds()
         {
             // Arrange
-            var topicEndpoint = Configuration.GetValue<string>("Arcus:EventGrid:TopicEndpoint");
-            var endpointKey = Configuration.GetValue<string>("Arcus:EventGrid:EndpointKey");
             const string licensePlate = "1-TOM-337";
             const string expectedSubject = "/";
             var eventId = Guid.NewGuid().ToString();
             var @event = new NewCarRegistered(eventId, licensePlate);
             var rawEventBody = JsonConvert.SerializeObject(@event.Data);
 
-            // Act
-            await EventGridPublisherBuilder
-                .ForTopic(topicEndpoint)
-                .UsingAuthenticationKey(endpointKey)
-                .Build()
-                .PublishRawAsync(@event.Id, @event.EventType, rawEventBody);
+            IEventGridPublisher publisher = _endpoint.BuildPublisher();
 
+            // Act
+            await publisher.PublishRawEventGridEventAsync(@event.Id, @event.EventType, rawEventBody);
             TracePublishedEvent(eventId, @event);
 
             // Assert
-            var receivedEvent = _serviceBusEventConsumerHost.GetReceivedEvent(eventId);
+            var receivedEvent = _endpoint.ServiceBusEventConsumerHost.GetReceivedEvent(eventId);
             AssertReceivedNewCarRegisteredEvent(eventId, @event.EventType, expectedSubject, licensePlate, receivedEvent);
         }
 
         [Fact]
         public async Task PublishSingleRawEventWithDetailedInfo_WithBuilder_ValidParameters_Succeeds()
         {
             // Arrange
-            var topicEndpoint = Configuration.GetValue<string>("Arcus:EventGrid:TopicEndpoint");
```

Comment:
Why was this test removed?

Reply:
We can't publish `RawEvent`'s anymore, like we discussed.